backend/api/views.py

```diff
@@ -25,9 +25,28 @@ def experiments(request):
             'slug': d.slug,
             'name': d.name,
             'enabled': d.enabled,
+            'realtime': False,
             'creationDate': d.created_at.date().isoformat() if d.created_at else None,
         })
 
+    # Include real-time experiments in list until they are fully imported.
+    imported = {d['slug'] for d in data}
+    experiments = list(
+        Enrollment.objects.exclude(experiment__in=imported)
+                          .distinct('experiment')
+                          .values_list('experiment', flat=True)
+    )
+    if experiments:
+        for exp in experiments:
+            data.append({
+                'id': None,
+                'slug': exp,
+                'name': None,
+                'enabled': True,
+                'realtime': True,
+                'creationDate': datetime.date.today().isoformat(),
```

Comment:
@spasovski Let me know if any of these defaults will break the front-end?

Reply:
Is it possible to grab the experiment name? If not I can replace with slug for now but it would look visually inconsistent. The rest seems fine.

Reply:
I don't really have a place to grab the name. It's not available until the import runs.